package.json

```diff
@@ -1,10 +1,14 @@
 {
   "name": "@terrestris/d3-util",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "description": "A set of helper classes for working with d3 charts.",
   "main": "src/index.js",
   "scripts": {
-    "release": "np --no-yarn"
+    "release": "np --no-yarn",
```

Comment:
You might also want to directly push tags here, so npm and github are in sync

Reply:
Thanks, added this in 0fc5684